components/v1/MarketDetailPage/LeveragedTokenMyAssetCard.tsx

```diff
@@ -35,32 +42,20 @@ const MyAssetCard: FunctionComponent<MyAssetCardProps> = ({ address }) => {
     const showError = navResponse.error || balanceResponse.error ? true : false;
     const showData = !showLoading && !showError && navResponse.data && balanceResponse.data ? true : false;
-
     return (
         <div className="flex w-full flex-col space-y-6 rounded-[16px] bg-gray-light-2 px-4 pb-4 dark:bg-gray-dark-2">
             <div className="pt-4">
                 <h2 className="text-base font-bold leading-4 text-gray-light-12 dark:text-gray-dark-12">My Asset</h2>
             </div>
-            <div className="flex flex-col space-y-6">
-                <div className="flex flex-row justify-between">
-                    <p className="text-sm leading-4 text-gray-light-10 dark:text-gray-dark-10">Balance</p>
-                    {(showLoading || showError) && <p className="h-[16px] w-[100px] animate-pulse rounded-[8px] bg-gray-light-3 dark:bg-gray-dark-3"></p>}
-                    {showData && (
-                        <p className="font-ibm text-sm font-semibold leading-4 tracking-[-.02em] text-gray-light-12 dark:text-gray-dark-12">
-                            {tokenBalanceFormatter.format(balance)} {metadata.title}
-                        </p>
-                    )}
-                </div>
-                <div className="flex flex-row justify-between">
-                    <p className="text-sm leading-4 text-gray-light-10 dark:text-gray-dark-10">Total value</p>
-                    {(showLoading || showError) && <p className="h-[16px] w-[100px] animate-pulse rounded-[8px] bg-gray-light-3 dark:bg-gray-dark-3"></p>}
-                    {showData && (
-                        <p className="font-ibm text-sm font-semibold leading-4 tracking-[-.02em] text-gray-light-12 dark:text-gray-dark-12">
-                            {tokenBalanceFormatter.format(balance * nav)} {metadata.debtSymbol}
-                        </p>
-                    )}
-                </div>
+            <div className="grid grid-cols-2 gap-8">
+                <AssetsItem title="Token Balance" image="/markets/tokenBalanceIcon.svg" value={`${tokenBalanceFormatter.format(balance)}`} showData={showData} showLoading={showLoading || showError} />
+                <AssetsItem title="Value (USDC)" image="/markets/valueIcon.svg" value={`${tokenBalanceFormatter.format(balance * nav)}`} showData={showData} showLoading={showLoading || showError} />
+                <AssetsItem title="Return" image="/markets/returnIcon.svg" value={`-`} showData={showData} showLoading={showLoading || showError} />
+                <AssetsItem title="Return (USDC)" image="/markets/returnDollarIcon.svg" value={`-`} showData={showData} showLoading={showLoading || showError} />
             </div>
+            <ButtonTertiary full onClick={async () => await addTokenToMetamask({ token: tokenType.ETHRISE, chainID: chain.chain.id })}>
+                Add {isVault ? metadata.vaultTitle : metadata.title}
+            </ButtonTertiary>
```

Comment:
1. The `isVaultToken` arguments is missing when calling `addTokenToMetamask` function here, when we press `Add rvETHUSDC` button, it's still adding `ETHRISE` to Metamask.
2. I think the text inside the button should be `Add {$TOKEN} to Wallet`

Reply:
ahh ya I forgot to dynamically change token.